torch/_inductor/cudagraph_trees.py

```diff
@@ -924,6 +933,23 @@ def _copy_input(self, idx, dst, src):
         # TODO - one jit kernel across multiple inputs
         dst.copy_(src)
 
+    def check_static_inputs_are_stable(self, new_inputs):
```

Comment:
This is never being invoked currently. Intentional?

Reply:
Nope, let me uncomment below